utils/src/treev2/quickPickWizard/FindByIdQuickPickStep.ts

```diff
@@ -15,15 +16,14 @@ interface FindByIdQuickPickOptions extends SkipIfOneQuickPickOptions {
 
 export class FindByIdQuickPickStep<TContext extends types.QuickPickWizardContext> extends GenericQuickPickStep<TContext, FindByIdQuickPickOptions> {
     public constructor(tdp: vscode.TreeDataProvider<unknown>, options: FindByIdQuickPickOptions) {
-        super(
-            tdp,
-            {
-                ...options,
-                skipIfOne: true, // Find-by-id is always skip-if-one
-            }
-        );
+        super(tdp, {
+            ...options,
+            skipIfOne: true, // Find-by-id is always skip-if-one
+        });
     }
 
+    readonly pickFilter: PickFilter = new FindByIdPickFilter(this.pickOptions);

```

Comment:
Since this isn't being called within the constructor, I'm not 100% sure that `this.pickOptions` will be available. I don't know the order for sure but I thought that class-scoped initializers like this get called _before_ the constructor.